test/tests.test.ts

```diff
@@ -84,11 +105,15 @@ describe('build', () => {
         server.httpServer.close()
       })
 
-      for (const { name, src, dest, transformedContent } of tests) {
+      for (const { name, src, dest, transformedContent, encoding } of tests) {
         test.concurrent(name, async () => {
-          const actual = await fetchTextContent(server, dest)
-          const expected = src === null ? null : await loadFileContent(src)
-          expect(actual).toBe(transformedContent ?? expected)
+          const expected =
+            src === null ? null : await loadFileContent(src, encoding)
```

Comment:
`expected` is `Buffer | string`, but I think it needs to be `ArrayBuffer | string`.

Reply:
Yeah indeed, but I can't find a way to convert it in a way that produces a valid ArrayBuffer :-/

Reply:
ok I found a package that does a good conversion